server/src/main/java/com/saralein/server/authorization/Authorizer.java

```diff
@@ -0,0 +1,34 @@
+package com.saralein.server.authorization;
+
+import com.saralein.server.request.Request;
+
+import java.util.Base64;
+
+public class Authorizer {
```

Comment:
This is coupled to a `Request` type, which makes it feel like a controller mix-in. Passing the encoded string from the header into `isAuthorized` instead of the whole request would make it more independent.

Other than that, this is a great abstraction. If we ever needed to authorize 2+ users, it'd be very easy/cheap to break out an interface and make it happen.